views/guide.lavender

```diff
@@ -297,10 +297,13 @@ block content
 
                 if FALSE
                   div this isn't rendered
+
                 elseif FALSE
```

Comment:
I added the space for readability. Probably not a good idea! I have not been able to get elseif to work at all (see Issue #53). I would be curious to know who has so I can figure out what I am doing wrong!